tutorials/README.md

```diff
@@ -22,3 +22,8 @@ An important focus of Cornac is making it convenient to work with auxiliary data
 - [Working with auxiliary data](./working_with_auxiliary_data.md)
 - [Text to Graph transformation](./text_to_graph.ipynb)
 - [Visual recommendation algorithm with text data](./vbpr_text.ipynb)
+
+## Stratified Evaluation
```

Comment:
Propensity Stratified Evaluation?
I think we should be consistent in the name of the method

Reply:
Good spot @tqtg --updated